File: Python-MySQL - Without Docker/app/files/models/report_status.py
from files import db


class ReportStatus(db.Model):
    __tablename__ = 'report_status'
    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
    description = db.Column(db.String(50), nullable=False)
    reports = db.relationship(
        "Report", backref="report_status")

    def __repr__(self):
        return f"{{\"id\": \"{self.id}\", \"description\": \"{self.description}\"}}"

File: Node-MySQL/app/models/ReportStatus.js
'use strict';
const {
    Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
    class ReportStatus extends Model {
        /**
         * Helper method for defining associations.
         * This method is not a part of Sequelize lifecycle.
         * The `models/index` file will call this method automatically.
         */
        static associate(models) {
            // define association here
        }
    }
    ReportStatus.init({
        id: {
            type: DataTypes.INTEGER,
            autoIncrement: true,
            primaryKey: true
        },
        description: {
            type: DataTypes.STRING(50),
            allowNull: false,
            validate: {
                notNull: { msg: "Descreption required!" }
            }
        }
    }, {
        sequelize,
        modelName: 'ReportStatus',
        tableName: 'ReportStatus',
        timestamps: false
    });
    return ReportStatus;
};